platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheCompositeBuildsIntegrationTest.groovy

```diff
@@ -23,7 +23,7 @@ import spock.lang.Issue
 
 class ConfigurationCacheCompositeBuildsIntegrationTest extends AbstractConfigurationCacheIntegrationTest {
 
-    def "can publish build scan with composite build"() {
+    def "can publish Build Scan with composite build"() {
```

Comment:
🤔 Should we really change test names as well?

Reply:
Consistent, since we are already at it, but it causes some sort of trouble, I'm fine not changing it.